lib/Toolbar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import icons from './icons';
+
+class Toolbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { showHeaders: false };
+  }
+
+  render() {
+    return (<div id="quill-toolbar">
+      <div className="ql-formats">
+        <button
+          className="ql-show-headers"
+          onClick={() => this.setState({ showHeaders: !this.state.showHeaders })}
+        >
+          { icons.aa() }
+        </button>
+        <div className={`ql-select-headers ${this.state.showHeaders ? 'show-headers' : ''}`}>
```

Comment:
What do you think to `showHeadings` over `showHeaders`?

Reply:
Or better yet, I prefer `chooseHeadings`, what do you think?